### DISCUSSION TITLE
Add special registers `*` and `+`

### DISCUSSION THREAD

**Discussion by hovsater** [helix-editor/helix#1519] [Ideas]
I'd like to propose two new special registers, `*` and `+` for manipulating primary (X11) and clipboard respectively. In https://github.com/helix-editor/helix/pull/310 integration with the system clipboard was added. In https://github.com/helix-editor/helix/pull/548 integration with the primary clipboard was added.

There are however some things we would need to take into consideration as these registers would not behave like other Helix registers when it comes to multiple selections.  [This comment](https://github.com/helix-editor/helix/pull/310#issuecomment-864362244) by @CBenoit explains this in more detail. The main selling point for supporting these registers would be the ability to use any motions that interact with a register (For instance, `xd` to delete the current line) and have it play nicely with either the primary or system clipboard.

@archseer and I already had a brief chat about this on Matrix, but I think it's worth getting input from as many people as possible to make sure it's a sound addition to Helix itself. 🙂

**Comment by goyalyashpal** [comment]
hi! is this solved?
